src/core/pusher.ts

```diff
@@ -88,7 +93,7 @@ export default class Pusher {
       level: TimelineLevel.INFO,
       version: Defaults.VERSION
     });
-    if (!this.config.disableStats) {
+    if (this.config.enableStats) {
```

Comment:
If we want to preserve backwards compatibility, we could change this to

    if (this.config.enableStats || (this.config.enableStats === undefined && this.config.disableStats === false)) {

Reply:
This should lead to the same behaviour. I think it's nicer

https://github.com/pusher/pusher-js/pull/407/commits/6e53ba4c1b94b8579e610256b57eba6a7138fc2c